module/Database/src/Form/Member.php

```diff
@@ -328,10 +332,33 @@ public function getInputFilterSpecification(): array
                 'iban' => [
                     'required' => true,
                     'validators' => [
-                        ['name' => Iban::class],
+                        [
+                            'name' => Iban::class,
+                            'options' => [
+                                'allow_non_sepa' => false,
+                                'messages' => [
+                                    Iban::NOTSUPPORTED     =>
+                                        $this->translator->translate('IBAN starts with unknown country code.'),
+                                    Iban::SEPANOTSUPPORTED =>
+                                        $this->translator->translate('IBANs from countries that do not participate in SEPA are not supported.'),
+                                    Iban::FALSEFORMAT      =>
+                                        $this->translator->translate('The input has a false IBAN format.'),
+                                    Iban::CHECKFAILED      =>
+                                        $this->translator->translate('The input has failed the IBAN check.'),
```

Comment:
Technically, I think we can leave out these translations as the default messages (and their translations) suffice imho.

Reply:
I misread your comment, thought the messages would not be translated, but indeed they are already automatically translated